### PR TITLE
MailSteward 17.1.1 (New Cask)

### PR DESCRIPTION
**Pull request by ghost**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [X] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [X] `brew audit --cask --online <cask>` is error-free.
- [X] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [X] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [X] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [X] `brew audit --cask --new <cask>` worked successfully.
- [X] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [X] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/m/mailsteward.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "mailsteward" do
+  version "17.1.1"
+  sha256 :no_check
+
+  url "https://s3.amazonaws.com/mailsteward/mailsteward.dmg",
+      verified: "s3.amazonaws.com/mailsteward/"
+  name "MailSteward"
+  desc "Ultimate E-mail Manager"
```
Comment (bevanjkay) [inline Casks/m/mailsteward.rb:8]:
```suggestion
  desc "Email management tool for Apple Mail and Postbox"

File: Casks/m/mailsteward.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  homepage "https://mailsteward.com/"
+
+  livecheck do
+    url "https://mailsteward.com/download/"
+    regex(/VERSION\s(\d+.\d+.\d+)/i)
```
Comment (bevanjkay) [inline Casks/m/mailsteward.rb:13]:
```suggestion
    url "https://mailsteward.com/upgrade.xml"
    strategy :sparkle, &:short_version

File: Casks/m/mailsteward.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+cask "mailsteward" do
+  version "17.1.1"
+  sha256 :no_check
+
+  url "https://s3.amazonaws.com/mailsteward/mailsteward.dmg",
```
Comment (bevanjkay) [inline Casks/m/mailsteward.rb:5]:
```suggestion
  url "https://s3.amazonaws.com/mailsteward/images/MailSteward_#{version}.zip",

File: Casks/m/mailsteward.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
+cask "mailsteward" do
+  version "17.1.1"
+  sha256 :no_check
```
Comment (bevanjkay) [inline Casks/m/mailsteward.rb:3]:
```suggestion
  sha256 "2cd47c8b3d9f5ab9d2ef6ee08b2a6d44aa64463297952e1f0ef5538c891daa2d"

File: Casks/m/mailsteward.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  name "MailSteward"
+  desc "Ultimate E-mail Manager"
+  homepage "https://mailsteward.com/"
+
+  livecheck do
+    url "https://mailsteward.com/download/"
+    regex(/VERSION\s(\d+.\d+.\d+)/i)
+  end
+
+  depends_on macos: ">= :catalina"
```
Comment (bevanjkay) [inline Casks/m/mailsteward.rb:16]:
```suggestion
  auto_updates true
  depends_on macos: ">= :catalina"

**Review by khipp — APPROVED** [review]
Thank you for your first contribution to `homebrew-cask`, @vodium-vastro! 🎉

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**